pyradiosky/skymodel.py

```diff
@@ -959,10 +1031,209 @@ def copy(self):
         """Overload this method to filter ra/dec warnings that shouldn't be issued."""
         # this method should be removed in version 0.3.0
         with warnings.catch_warnings():
-            warnings.filterwarnings("ignore", message="ra is no longer")
-            warnings.filterwarnings("ignore", message="dec is no longer")
+            warnings.filterwarnings("ignore", message="lon is no longer")
+            warnings.filterwarnings("ignore", message="lat is no longer")
             return super(SkyModel, self).copy()
 
+    def transform_to(self, frame):
+        """Transform to a difference coordinate frame using underlying Astropy function.
+
+        This function is a thin wrapper on astropy.coordinates.SkyCoord.transform_to
+        please refer to that function for full documentation.
+
+        Parameters
+        ----------
+        frame : str, `BaseCoordinateFrame` class or instance.
+            The frame to transform this coordinate into.
+            Currently frame must be one of ["galactic", "icrs"].
+
+
+        """
+        if self.component_type == "healpix":
+            raise ValueError(
+                "Direct coordinate transformation between frames is not valid"
+                " for `healpix` type catalogs. Please use the `healpix_interp_transform` "
+                "to transform to a new frame and interpolate to the new pixel centers."
```

Comment:
You could add "or use `healpix_to_point` to convert to point sources before transforming frames.`"